homeassistant/components/envisalink/services.yaml

```diff
@@ -0,0 +1,15 @@
+# Describes the format for available for Envisalink services.
+
+invoke_custom_function:
+  description: >
+    Allows users with DSC panels to trigger a PGM output (1-4).
+    Note that you need to specify the alarm panel's "code" parameter for this to work.
+  fields:
+    partition:
+      description: >
+        The alarm panel partition to trigger the PGM output on.
+        Typically this is just "1".
+      example: "1"
+    pgm:
+      description: The PGM number to trigger on the alarm panel.  This will be 1-4.
```

Comment:
Remove double space.

Reply:
Sure will do- just so I understand, you're saying the description/example tags should be in line with hte parametr names?
so:
```
pgm:
description: description here
```

Reply:
Great thanks- all set!